data/api/client-server/profile.yaml

```diff
@@ -69,6 +69,10 @@ paths:
                 
                 For `displayname`, the value MUST be a string.
                 
+                For `m.tz`, the value MUST be a valid identifier from the  [IANA Time Zone Database](https://www.iana.org/time-zones).
+                Servers MAY choose to validate the value. Clients MUST expect unknown or invalid
+                values.
```

Comment:
Not sure if these options should be moved off to another part of the spec, but also didn't want to block that on getting the feature in.